raft/log.go

```diff
@@ -791,6 +827,7 @@ loop:
 	// We don't add the +1 because the slice starts from 0.
 	quorumIndex := (nodeN / 2)
 	if quorumIndex >= len(indexes) {
+		l.tracef("sending heartbeat: no quorum: n=%d", quorumIndex)
```

Comment:
D'oh -- I assumed that `break loop` meant break and jump out to loop, but checking the spec it I see it means break the loop indicated by the label. OK, mystery solved.